bitnami/external-dns/templates/deployment.yaml

```diff
@@ -618,6 +641,12 @@ spec:
             path: {{ .Values.designate.customCAHostPath }}
         {{- end }}
         {{- end }}
+        {{- if and (eq .Values.provider "rfc2136") .Values.rfc2136.rfc3645Enabled }}
+        # Kerberos config volume
+        - name: krb5config
+          configMap:
+            name: {{ template "external-dns.fullname" . }}
+        {{- end }}
```

Comment:
We should probably specify only the key and path we want from the Config Map, just as we do with the CA one.

Reply:
Done in latest commits